addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -175,12 +175,23 @@
 
         @include media-only(print) {
             font-size: 0.8rem;
+            page-break-inside: avoid;
         }
 
         &:not(.o_kanban_ghost) {
             padding: var(--KanbanRecord-padding-v) var(--KanbanRecord-padding-h);
             border: $border-width solid $border-color;
             background-color: $o-view-background-color;
+
+            @include media-only(print) {
+                & > *, & > * > * {
+                    height: auto;
+                }
```

Comment:
This selector gives me the creeps :grimacing: :sweat_smile: 

Seriously, though, I'm wondering if its main purpose isn't to counter the `height: 100%` set on `.o_kanban_renderer .o_kanban_record:not(.o_legacy_kanban_record) > main` and if this `height` rule is actually needed ?